modules/swagger-codegen/src/test/java/io/swagger/codegen/AbstractIntegrationTest.java

```diff
@@ -21,7 +21,7 @@
     protected abstract Map<String, String> configProperties();
 
     // @wing328: ignore for the time being until we fix the error with the integration test
-    @Test(enabled = false)
+    @Test(enabled = true)
```

Comment:
If my memory serves me well, the test failed to run on Windows and that's why I commented it out for the time being until some Windows developers have a chance to fix it.